paimon-spark/paimon-spark-common/src/test/java/org/apache/paimon/spark/SparkWriteITCase.java

```diff
@@ -237,4 +237,36 @@ public void testNonnull() {
             spark.sql("DROP TABLE IF EXISTS S");
         }
     }
+
+    @Test
+    public void testShowPartitionsPartitioned() {
```

Comment:
Maybe put these in a new test class named `ShowCommandTest` which will contain case related to `SHOW` commands.